pom.xml

```diff
@@ -96,6 +105,18 @@ Copyright 2014 CyberVision, Inc.
         <sonar.skippedModules>admin,admin-rest-client,client-java-android,gwt-client</sonar.skippedModules>
         <sonar.jacoco.reportPath>${main.dir}/target/jacoco.exec</sonar.jacoco.reportPath>
         <sonar.jacoco.itReportPath>${main.dir}/target/jacoco-it.exec</sonar.jacoco.itReportPath>
+
+        <android.version>2.2.1</android.version>
+
+        <javax.mail.version>1.4</javax.mail.version>
+        <javax.websocket.version>1.0</javax.websocket.version>
+
```

Comment:
as I understood these blank lines are used to logically divide different properties